src/components/common/Directors.vue

```diff
@@ -795,36 +821,19 @@ export default class Directors extends Mixins(CommonMixin, DateMixin, DirectorMi
     }
 
     // cessation date must be after appointment date
-    const rule1 =
-      v => this.compareDates(v, appointmentDate, '>') || 'Cessation Date must be after Appointment Date'
-
-    rules.push(rule1)
+    rules.push(v => this.compareDates(v, appointmentDate, '>') || 'Cessation Date must be after Appointment Date')
 
     // cessation date must be in the past (or today)
-    const rule2 =
-      v => this.dateIsNotFuture(v) || 'Cessation Date cannot be in the future'
-
-    rules.push(rule2)
+    rules.push(v => this.dateIsNotFuture(v) || 'Cessation Date cannot be in the future')
 
     return rules
   }
 
-  /**
-   * Called when component is mounted.
-   */
-  private mounted (): void {
```

Comment:
This never did anything because the As Of Date wasn't set at this point in time, so the directors never got fetched.